planprogenerator/generator.py

```diff
@@ -30,23 +30,19 @@ def generate(self, nodes, edges, signals, filename=None):
         self.control_elements = []
         self.trips = []
         self.routes = []
+        self.config = config
 
         # Create Trip
         trip = Trip(edges)
         for signal in signals:
             signal.trip = trip
 
-        # Create Routes
-        route_generator = RouteGenerator(nodes, edges, signals)
-        routes = route_generator.generate_routes()
-
         self.uuids = self.uuids + RootXML.get_root_uuids()
 
         self.generate_nodes(nodes)
         self.generate_edges(edges)
         self.generate_signals(signals)
         self.generate_trips([trip])
-        self.generate_routes(routes)
```

Comment:
This lines needs to stay in here, otherwise the Fahrstraßen will not be part of the PlanPro file